fuel-tx/src/transaction.rs

```diff
@@ -500,6 +500,9 @@ pub mod field {
 
         /// Returns predicate's offset and length of the `Input` at `idx`, if any.
         fn inputs_predicate_offset_at(&self, idx: usize) -> Option<(usize, usize)>;
+
+        /// Returns predicate's offset and length of the `Input` at `idx`, if any.
+        fn inputs_predicate_gas_used_at(&self, idx: usize) -> Option<Word>;
```

Comment:
Yeh this comment indicates that it's a memory offset not a value

Reply:
removed the function and refactored